src/modality-agnostic-files.md

```diff
@@ -362,20 +362,63 @@ If the dataset includes multiple sets of participant level measurements (for
 example responses from multiple questionnaires) they can be split into
 individual files separate from `participants.tsv`.
 
-Each of the measurement files MUST be kept in a `/phenotype` directory placed
+Each of the measurement files MUST be kept in a `phenotype/` directory placed
 at the root of the BIDS dataset and MUST end with the `.tsv` extension.
 Filenames SHOULD be chosen to reflect the contents of the file.
 For example, the "Adult ADHD Clinical Diagnostic Scale" could be saved in a file
-called `/phenotype/acds_adult.tsv`.
+called `phenotype/acds_adult.tsv`.
 
 The files can include an arbitrary set of columns, but one of them MUST be
 `participant_id` and the entries of that column MUST correspond to the subjects
 in the BIDS dataset and `participants.tsv` file.
 
+!!! success "Guideline 2"
+
+    For [best tabular phenotypic data](./appendices/phenotype.md):
+    It is REQUIRED to aggregate all participant data into
+    one TSV per tabular phenotypic file.
+
+In phenotypic and assessment data each measurement tool has
```

Comment:
Not sure where this goes, but I would also say we elevate `MeasurementToolMetadata` from currently `OPTIONAL` to `RECOMMENDED` because that is where you would explain what the tool is actually about (and link to a controlled term) - which will improve the reusability a lot